src/helpers/misc.ts

```diff
@@ -274,3 +274,35 @@ export function deepEquals(o1: any, o2: any): boolean {
 
   return true;
 }
+
+/**
+ * Alternative to Math.max that works with large arrays.
+ * Typically useful for arrays bigger than 100k elements.
+ */
+export function largeMax(array: any[]) {
```

Comment:
why `any` ? If we type this correctly, we can remove `Number(...)` below, which inevitably eats a bit of performance.
```suggestion
export function largeMax(array: number[]) {
```

Reply:
I'll move the number out of the loop then ? I wanted to make this wrapper as "close" to number as possible, and it supports strings as well (at the very least)

Reply:
Accepting everything (and n'importe quoi) to get a random result is one of the bad parts of javascript though IMHO. I would not reproduce this (bad) behavior.

With numbers, it's crystal clear what's the result of 'max'. The max of strings however... is the result the last in the alphabetical order? is it the lengthier string?